### ISSUE TITLE
Can't open the app

### ISSUE THREAD

**Issue by shuvashish76** [thunderbird/thunderbird-android#7988]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

6.902 (39002)

### Where did you get the app from?

F-Droid

### Android version

9 (LineageOS unofficial)

### Device model

Lenovo Tab4 8 plus

### Steps to reproduce

Just opening the app crashes

### Expected behavior

Open normally

### Actual behavior

Crashes

### Logs

```
FATAL EXCEPTION: main
Process: com.fsck.k9, PID: 10578
java.lang.RuntimeException: Unable to resume activity {com.fsck.k9/com.fsck.k9.activity.MessageList}: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.TextView.setText(java.lang.CharSequence)' on a null object reference
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3822)
	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3854)
	at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:51)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1816)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6718)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.TextView.setText(java.lang.CharSequence)' on a null object reference
	at com.fsck.k9.activity.MessageList.setActionBarTitle(MessageList.kt:937)
	at com.fsck.k9.activity.MessageList.setMessageListTitle(MessageList.kt:951)
	at com.fsck.k9.ui.messagelist.MessageListFragment.setWindowTitle(MessageListFragment.kt:471)
	at com.fsck.k9.ui.messagelist.MessageListFragment.updateTitle(MessageListFragment.kt:434)
	at com.fsck.k9.ui.messagelist.MessageListFragment.onResume(MessageListFragment.kt:591)
	at androidx.fragment.app.Fragment.performResume(Fragment.java:3206)
	at androidx.fragment.app.FragmentStateManager.resume(FragmentStateManager.java:658)
	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:302)
	at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:114)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1663)
	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3247)
	at androidx.fragment.app.FragmentManager.dispatchResume(FragmentManager.java:3179)
	at androidx.fragment.app.FragmentController.dispatchResume(FragmentController.java:285)
	at androidx.fragment.app.FragmentActivity.onResumeFragments(FragmentActivity.java:333)
	at androidx.fragment.app.FragmentActivity.onPostResume(FragmentActivity.java:322)
	at androidx.appcompat.app.AppCompatActivity.onPostResume(AppCompatActivity.java:245)
	at android.app.Activity.performResume(Activity.java:7325)
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3814)
	... 11 more
```

**Comment by pxcask** [comment]
Same here since this morning on my FP4 running v6.902 on eOS 2.1.t 20240603[...] stable.
I can't even open the app. It closes immediately. I emptied the cache, restarted the phone, However, the app is visible in the overview screen of running apps showing its K9-logo only on a black background unless I force-quit it.

**Comment by shuvashish76** [comment]
Just installed F-Droid v6.903 (39003), still crashes.
<details><summary>Logs:</summary>
<p>

```
FATAL EXCEPTION: main
Process: com.fsck.k9, PID: 4404
java.lang.RuntimeException: Unable to resume activity {com.fsck.k9/com.fsck.k9.activity.MessageList}: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.TextView.setText(java.lang.CharSequence)' on a null object reference
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3822)
	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3854)
	at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:51)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1816)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6718)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.TextView.setText(java.lang.CharSequence)' on a null object reference
	at com.fsck.k9.activity.MessageList.setActionBarTitle(MessageList.kt:937)
	at com.fsck.k9.activity.MessageList.setMessageListTitle(MessageList.kt:951)
	at com.fsck.k9.ui.messagelist.MessageListFragment.setWindowTitle(MessageListFragment.kt:471)
	at com.fsck.k9.ui.messagelist.MessageListFragment.updateTitle(MessageListFragment.kt:434)
	at com.fsck.k9.ui.messagelist.MessageListFragment.onResume(MessageListFragment.kt:591)
	at androidx.fragment.app.Fragment.performResume(Fragment.java:3206)
	at androidx.fragment.app.FragmentStateManager.resume(FragmentStateManager.java:658)
	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:302)
	at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:114)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1663)
	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3247)
	at androidx.fragment.app.FragmentManager.dispatchResume(FragmentManager.java:3179)
	at androidx.fragment.app.FragmentController.dispatchResume(FragmentController.java:285)
	at androidx.fragment.app.FragmentActivity.onResumeFragments(FragmentActivity.java:333)
	at androidx.fragment.app.FragmentActivity.onPostResume(FragmentActivity.java:322)
	at androidx.appcompat.app.AppCompatActivity.onPostResume(AppCompatActivity.java:245)
	at android.app.Activity.performResume(Activity.java:7325)
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3814)
	... 11 more
```
</p>
</details>

**Comment by xhajt03** [comment]
Indeed, still crashing.

**Comment by cketti** [comment]
The bug is only fixed in K-9 Mail 6.904.
